examples/softirq-latency-net-rx.bpf.c

```diff
@@ -86,7 +96,7 @@ int BPF_PROG(softirq_raise, unsigned int vec_nr)
 SEC("tp_btf/softirq_entry")
 int BPF_PROG(softirq_entry, unsigned int vec_nr)
 {
-    u64 delta_us, *raise_ts_ptr, *existing_entry_ts_ptr, *serviced_total_ptr, ts;
+    u64 delta_ns, *raise_ts_ptr,  *serviced_total_ptr, ts;
```

Comment:
The failed `clang-format` build has the diff that needs to be applied to make it happy.

Reply:
Used clang-format and fixed this and others